src/ppl/src/IOPlacer.cpp

```diff
@@ -727,9 +727,46 @@ Edge IOPlacer::getMirroredEdge(const Edge& edge)
   return mirrored_edge;
 }
 
+int IOPlacer::computeRegionIncrease(const Interval& interval, int num_pins)
+{
+  bool vertical
+      = interval.getEdge() == Edge::top || interval.getEdge() == Edge::bottom;
```

Comment:
Do you mean the pins are vertical?  The edges themselves are horizontal.  Similarly in IOPlacer::findSlots